integration/combination/test_function_with_application.py

```diff
@@ -0,0 +1,17 @@
+from integration.helpers.base_test import BaseTest
+
+
+class TestFunctionWithApplication(BaseTest):
+    def test_function_referencing_outputs_from_application(self):
+        self.create_and_verify_stack("combination/function_with_application")
+
+        lambda_funciton_name = self.get_physical_id_by_type("AWS::Lambda::Function")
```

Comment:
nit: little type `funciton` -> `function`

Reply:
Oops, will fix it.